doc/user/sql/create-sink.md

```diff
@@ -26,22 +26,28 @@ Field | Use
 **IF NOT EXISTS** | If specified, _do not_ generate an error if a sink of the same name already exists. <br/><br/>If _not_ specified, throw an error if a sink of the same name already exists. _(Default)_
 _sink&lowbar;name_ | A name for the sink. This name is only used within Materialize.
 _item&lowbar;name_ | The name of the source or view you want to send to the sink.
+**FILE** _path-prefix_ | The absolute path and file name prefix of file to create and write to.
+
+### Kafka connector
+
+{{< diagram "kafka-connector.html" >}}
+
+Field | Use
+------|-----
 **KAFKA BROKER** _host_ | The Kafka broker's host name.
-**TOPIC** _topic_ | The Kafka topic to write to.
+**TOPIC** _topic-prefix_ | The prefix used to generate the Kafka topic name to create and write to.
 **CONFLUENT SCHEMA REGISTRY** _url_ | The URL of the Confluent schema registry to get schema information from.
-**SCHEMA FILE** _path_ | The absolute path to a file containing the schema.
-**SCHEMA** _inline&lowbar;schema_ | A string representing the schema.
 
 ## Detail
 
-- Materialize currently only supports Kafka sinks.
+- Materialize currently only supports Avro formatted sinks that write to either a single partition topic or a Avro object container file.
+- On each restart, Materialize creates new, distinct topics and files for each sinks. Topic and file names are suffixed with {topic-name}-{sink global-id}-{startup-time}-{nonce}, and file names preserve the given extension.
+- Materialize stores information about actual topic names and actual file names in the `mz_kafka_sinks` and `mz_avro_ocf_sinks` log sources.
```

Comment:
Can I get you to link to the example here? I at first thought these would require a schema name but realize that isn't the case!